lib/sensu-handler.rb

```diff
@@ -209,6 +209,9 @@ def filter_dependencies
         if @event['check']['dependencies'].is_a?(Array)
           @event['check']['dependencies'].each do |dependency|
             begin
+              if(dependency == "")
```

Comment:
rubocop would want the 1 line style.    more people agree on those guidelines. 

the .empty?  would cover strings empty arrays and and empty hashes which, since the check data comes from arbitrary json it could be any of those.  (though it should only be strings)    people probably vary on that preference.  

it looks idomatic to _me_ :)